rosario_angelica/country.js

```diff
@@ -0,0 +1,64 @@
+const countryInput = document.getElementById('country-input');
```

Comment:
If you're done changing IDs in your county.html, do not forget to change also your ID naming in country.js, so that it will get the appropriate id. For example:
❌const countryInput = document.getElementById('country-input');
✅const countryInput = document.getElementById('country_input');